packages/components/src/tabs/styles.ts

```diff
@@ -123,7 +153,7 @@ export const Tab = styled( Ariakit.Tab )`
 		}
 
 		// Focus.
-		&::before {
+		&::after {
```

Comment:
Yes, it was messing with the styles in the inserter patterns/media vertical tabs. The z-index was a hack in the first place since we were using `::after` for something that should have a natural position _before_ the items that show in front.

Reply:
You can witness this problem yourself by testing this PR: https://github.com/WordPress/gutenberg/pull/65387

Style overrides are gone there and you can see there's no indicator, however if you swap before and after (remember the `transitionend` event), and remove the z-index you'll see it again.

Reply:
One final note for context is that I only picked `::after` for the indicator because `::before` was taken for the focus stuff to make the most minimal change. There was no other reason and I disliked the z-index from the start, knew something like this was bound to happen :P